scripts/prog15.rb

```diff
@@ -0,0 +1,18 @@
+def prime(n)
+    (2..n).step(1).each do |value|
+        c=0
```

Comment:
Also, instead of naming this variable `c`, we can use something more meaningful like `is_prime?`. Try to write the program such that for someone who is reading it, seems like plain english.